specifyweb/frontend/js_src/lib/components/Forms/Save.tsx

```diff
@@ -107,8 +108,14 @@ export function SaveButton<SCHEMA extends AnySchema = AnySchema>({
   const canUpdate = hasTablePermission(resource.specifyTable.name, 'update');
   const canSave = resource.isNew() ? canCreate : canUpdate;
 
+  const location = useLocation();
+  const currentUrl = location.pathname;
+  const isInViewSet = currentUrl.includes('view-set');
```

Comment:
```suggestion
  const isInFormEditor = React.useContext(InFormEditorContext);
```

see https://github.com/specify/specify7/blob/71ad206747904a98fac6043116e2c05331f5036c/specifyweb/frontend/js_src/lib/components/FormMeta/index.tsx#L58

Reply:
mhhh okok! Much better